androidApp/src/main/java/dev/johnoreilly/confetti/sessiondetails/SessionDetailsView.kt

```diff
@@ -19,13 +19,15 @@ import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.unit.dp
 import androidx.lifecycle.compose.collectAsStateWithLifecycle
 import com.google.accompanist.flowlayout.FlowRow
-import dev.johnoreilly.confetti.SessionsUiState
+import dev.johnoreilly.confetti.SessionDetailsViewModel
 import dev.johnoreilly.confetti.fragment.SessionDetails
-import dev.johnoreilly.confetti.ui.ConfettiScaffold
 import org.koin.androidx.compose.getViewModel
 
 @Composable
-fun SessionDetailsRoute(onBackClick: () -> Unit, viewModel: SessionDetailsViewModel = getViewModel()) {
+fun SessionDetailsRoute(conference: String, sessionId: String, onBackClick: () -> Unit) {
+    val viewModel: SessionDetailsViewModel = getViewModel<SessionDetailsViewModel>().apply {
+        configure(conference, sessionId)
+    }
```

Comment:
I'm not specifically happy with this pattern of passing arguments to a ViewModel. Ideally, we could compute those from `SavedStateHandle` directly in the VM but that won't work on iOS :(.